apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/question/PageQuestionDeleter.java

```diff
@@ -0,0 +1,38 @@
+package gov.cdc.nbs.questionbank.page.content.question;
+
+import gov.cdc.nbs.questionbank.entity.WaTemplate;
+import gov.cdc.nbs.questionbank.entity.question.WaQuestion;
+import gov.cdc.nbs.questionbank.page.command.PageContentCommand;
+import gov.cdc.nbs.questionbank.page.exception.DeleteQuestionException;
+import gov.cdc.nbs.questionbank.question.exception.QuestionNotFoundException;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import javax.persistence.EntityManager;
+import java.time.Instant;
+
+@Component
+@Transactional
+public class PageQuestionDeleter {
+
+    private final EntityManager entityManager;
+
+    public PageQuestionDeleter(
+            final EntityManager entityManager) {
+        this.entityManager = entityManager;
+    }
+
+    public void deleteQuestion(Long pageId,Long questionId, Long user) {
+
+        WaTemplate page = entityManager.find(WaTemplate.class, pageId);
+        if (page == null) {
+            throw new DeleteQuestionException("Failed to find page with id: " + pageId);
+        }
+        WaQuestion question = entityManager.find(WaQuestion.class,questionId);
+        if (question == null) {
+            throw new QuestionNotFoundException(questionId);
+        }
```

Comment:
Suggestion: This check probably isn't necessary since the same check is done within `WaTemplate`